### PR TITLE
Added tests for 13903

### PR DESCRIPTION
**Pull request by bfloreaDVSA**

### CODE REVIEW

File: src/main/java/steps/TestResultsSteps.java [hunk 1]
```diff
@@ -580,6 +580,13 @@ public void validateTestFieldExists(String fieldName) {
         assertThat(response.then().body("[0].testTypes[0]", hasKey(fieldName)));
     }
 
+    @Step
+    public void validateTestFieldNotPresent(String fieldName) {
+//        response.then().log().all();
```
Comment (ZacM2) [inline src/main/java/steps/TestResultsSteps.java:585]: remove commented out line

Comment (bfloreaDVSA) [reply src/main/java/steps/TestResultsSteps.java:585]: we'll keep these commented to easily uncomment for quick debugging purposes

**Review by KHodderDVSA — APPROVED** [review]
Assured it's good by Bogdan

**Review by ghost — APPROVED** [review]
Approving as above

**Pull request merged**